policy/README.md

```diff
@@ -0,0 +1,296 @@
+
+##### Evaluation Pseudocode
+
+The below example is intended to highlight the bucketing mechanisms of rule evaluation, and should not be considered a fully-fledged pseudocode representation of how to evaluate a policy. This is specifically for count maximum rules; evaluation for other rule types will be explained as part of the Compliance API.
+
+```
+let p = Policy object
+let rules = p.rules
+let S = set of vehicles to consider (e.g. all vehicles for a specific provider)
+
+eval(rules, S) {
+    let exclude = [] // Empty set
+    let matched_not_bucketed = [] // Empty set
+    for rule in rules {
+        let result = eval_rule(rule, S \ exclude)
+        let matched_vehicles = all (violation_vehicle || violation_vehicles) in result
+        let bucketed_vehicles = matched_vehicles(accumulator, vehicle => {
+            if (accumulator.length < rule.maximum) {
+                accumulator.add(vehicle)
+            }
+        })
+        matched_not_bucketed.add(matched_vehicles \ bucketed_vehicles)
+        exclude.add(bucketed_vehicles)
+        ...
+    }
+    ...
+}
+...
+```
+
+<a name="messages"></a>
+
+### Messages
+
+Some Policies as established by the Agency may benefit from rider communication. This optional field contains a map of languages to be (optionally?) shown to the user. (Unanswered question: shown with what frequency? At sign-up? Once per day?)
+
+Language tag values will be per [BCP 47](https://www.rfc-editor.org/rfc/bcp/bcp47.txt).
+
+Example for a decreased speed-limit rule for Venice Beach on weekends:
+
+```
+	"messages": {
+	            "en-US": "Remember to stay under 10 MPH on Venice Beach on weekends!”,
+	            "es-US": "¡Recuerda mantener por debajo 10 millas por hora en Venice Beach los fines de semana!"
+	        },
+```
+
+<a name="value-url"></a>
+
+### Value URL
+
+An Agency may wish to provide dynamic or global rules, e.g. "Within 300 yards of the stadium, 1000 total extra scooters may be deployed, across all Provider(s)." In this case, compliance is not computable from the information available to a single Provider. The Agency would provide a specified endpoint to get the current count of vehicles in the service-area, so that individual Providers could decide whether adding some number to those present can be done.
+
+There are potential complexities that are as-yet unaddressed, such as the latency between a Provider deciding to enter such an area, and the number of vehicles in the area when the scooters arrive. As this specification is Alpha, feedback and refinements are welcome.
```

Comment:
Now that this PR is close to being merged, what's the plan for aspects like this that are called out as not being fully baked yet?